ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml

```diff
@@ -1070,7 +1070,7 @@ tests:
         MIRROR_IMAGES=true
         INSTALLER_PROXY=true
         NETWORK_TYPE=OVNKubernetes
-      E2E_RUN_TAGS: '@amd64 and @baremetal-ipi and not @fips'
+      E2E_RUN_TAGS: '@amd64 and @baremetal-ipi and @disconnected and not @fips'
       TEST_ADDITIONAL: Network_Observability
```

Comment:
Please add bellow for ginkgo test.
```TEST_FILTERS: ~ChkUpgrade&;~NonPreRelease&;~Serial&;~Disruptive&;~ConnectedOnly&;~HyperShiftMGMT&;~MicroShiftOnly&```

for example:
https://github.com/openshift/release/blob/master/ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-nightly.yaml#L180C7-L180C7

Reply:
thanks. updated